src/plugins/copyCode.js

```diff
@@ -0,0 +1,77 @@
+const cheerio = module.parent.require('cheerio');
+
+const {
+  COPIED_TO_CLIPBOARD,
+  COPY_ICON,
+  COPY_TO_CLIPBOARD,
+} = require('../constants');
+
+function getButtonHTML() {
+  const html = `<button class="copy-btn-icon copy-btn" type="button">
+    <div class="copy-btn-body">
+    ${COPY_ICON}
+    <strong class="copy-btn-label">${COPY_TO_CLIPBOARD}</strong>
+    </div>
+    </button>`;
+  return html;
+}
+
+function getCopyCodeBlockScript() {
+  return `<script>
+    document.onclick = ((event) => {
+        let element = event.target;
+        const elementClassList = [...element.classList];
+        const listOfSelectors = ['copy-btn-label', 'copy-btn', 'copy-btn-icon', 'copy-btn-body', 'cpy'];
```

Comment:
This should be declared outside the `onClick` handler. Also, a more descriptive name (e.g. `copyButtonSelectors`) might help to explain what exactly these selectors are for.

Reply:
Removed it entirely since the click event listener is added to the `Copy` button.